src/org/rascalmpl/library/IO.rsc

```diff
@@ -113,9 +113,9 @@ Append a textual representation of some values to an existing or a newly created
 
 Files are encoded using the charset provided.
 }
-@javaClass{org.rascalmpl.library.Prelude}
-public java void appendToFileEnc(loc file, str charset, value V...)
-throws PathNotFound, IO;
+@deprecated{Use `appendToFile(file, charset="UTF-8")` instead.}
+public void appendToFileEnc(loc file, str charset, value V...) throws PathNotFound, IO
+  = appendToFile(file, V, charset=charset);
```

Comment:
Is this allowed on void functions?

Reply:
good question!

Reply:
Thinking about it, probably only if the called function returns `void`, otherwise the type-checker complains.